uv.lock

```diff
@@ -1,11 +1,15 @@
 version = 1
-revision = 1
```

Comment:
What version of `uv` did you generate this with? Probably want to do a `uv self update` because I can't read this diff 🙈